pkgs/development/web/nodejs/v16.nix

```
@@ -8,14 +8,7 @@ let
 in
   buildNodejs {
     inherit enableNpm;
-    version = "16.5.0";
-    sha256 = "16dapj5pm2y1m3ldrjjlz8rq9axk85nn316iz02nk6qjs66y6drz";
-    patches = [
-      # Fix CVE-2021-22930 https://github.com/nodejs/node/pull/39423.
-      # It should be fixed by Node.js 16.6.0, but currently it fails to build on Darwin
-      (fetchpatch {
-        url = "https://github.com/nodejs/node/commit/9d950a0956bf2c3dd87bacb56807f37e16a91db4.patch";
-        sha256 = "1narhk5dqdkbndh9hg0dn5ghhgrd6gsamjqszpivmp33nl5hgsx3";
-      })
-    ];
+    version = "16.6.1";
+    sha256 = "0mz5wfhf2k1qf3d57h4r8b30izhyg93g5m9c8rljlzy6ih2ymcbr";
+    patches = lib.optionals stdenv.isDarwin [ ./disable-darwin-v8-system-instrumentation.patch ];
```

```suggestion
    patches = [ ./disable-darwin-v8-system-instrumentation.patch ];
```
This shouldn't break anything I think.